libs/ECS/Components/WeightComponent.hpp

```diff
@@ -0,0 +1,32 @@
+/*
+** EPITECH PROJECT, 2022
+** Project
+** File description:
+** WeightComponent
+*/
+
+#ifndef WEIGHTCOMPONENT_HPP_
+#define WEIGHTCOMPONENT_HPP_
+
+#include "Component.hpp"
+
+namespace ecs
+{
+    /// @brief The Weight class, almost used for everything, used in gravity system
+    class Weight : public Component{
+      public:
+        /// @brief weight of the entity
+        unsigned short weight;
```

Comment:
A weight can't be negative ? Maybe it can be fun with the gravity

Reply:
That's okay for me ! I will change it